tools/build_variables.bzl

```diff
@@ -134,6 +134,7 @@ core_sources_full_mobile = [
     "torch/csrc/jit/api/function_impl.cpp",
     "torch/csrc/jit/api/module.cpp",
     "torch/csrc/jit/api/object.cpp",
+    "torch/csrc/jit/backends/backend.cpp",
     "torch/csrc/jit/backends/backend_detail.cpp",
```

Comment:
The functions are used in backend registration, which is also used in the mobile client. That's why I put it in core_sources_full_mobile, which is used to build mobile runtime. I assume that `_def_unboxed` is used to register functions in both jit and python, so that it's also needed from C++ runtime?